src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -317,15 +408,22 @@ private Vector limitDirectionSlewRate(Vector spdVec) {
     return (new Vector(m_currentTranslationMag, 0)).rot(m_currentTranslationDirRadians);
   }
 
+  /**
+   * checks whether pathplanner paths should be flipped based on the current alliance
+   *
+   * @return whether pathplanner paths should be flipped
+   */
+  private boolean allianceCheck() {
+    var alliance = DriverStation.getAlliance();
+    if (alliance.isPresent()) {
+      return alliance.get() == DriverStation.Alliance.Red;
+    }
+    return (false);
+  }
+
   /** Zeroes the heading of the robot. */
   public void zeroHeading() {
     m_headingOffsetRadians = getGyroAngle().in(Units.Radians);
     m_gyro.reset();
   }
-
-  /** run periodically when being simulated, required but not used in this implementation */
-  @Override
```

Comment:
Why remove this?

Reply:
it didn't do anything, and it turns out it didn't need to be there, so it was just clutter